apps/dashboard/app/views/projects/edit.html.erb

```diff
@@ -1,7 +1,20 @@
 <div class='page-header text-center'>
-  <h1>Edit Project</h1>
+  <h1>
+    Editing: <%= @project.name %>
+  </h1>
 </div>
 
 <br>
```

Comment:
Instead of having a line break here, let's give the div above some bottom padding with a `pb-` bootstrap padding class.

Reply:
I missed this on my last commit but will make the adjustment.